src/sanitizer.js

```diff
@@ -8,18 +8,67 @@
 // eslint-disable-next-line no-unused-vars
 export const _sanitizeDocFragment = function _sanitizeDocFragment(config, input) {
     // TODO implement this function and remove eslint-disable comment above
+    // TODO maybe we can use https://developer.mozilla.org/en-US/docs/Web/API/NodeIterator to traverse through nodes
```

Comment:
100%, yes :). Can you remove this comment and add it to https://github.com/mozilla/sanitizer-polyfill/issues/4 instead though? The current information in that issue is similar, but one-step removed. It points to DOMPurify code which also uses the NodeIterator interface.